go/vt/vttablet/tabletmanager/tm_state.go

```diff
@@ -181,7 +181,24 @@ func (ts *tmState) ChangeTabletType(ctx context.Context, tabletType topodatapb.T
 		// Update the tablet record first.
 		_, err := topotools.ChangeType(ctx, ts.tm.TopoServer, ts.tm.tabletAlias, tabletType, PrimaryTermStartTime)
 		if err != nil {
-			return err
+			log.Errorf("error in changing type in tablet record in topo for tablet %s :- %v", topoproto.TabletAliasString(ts.tm.tabletAlias), err)
```

Comment:
Sorry to be nitpicky, but I'm trying to rephrase the errors to be more concise. Maybe "error changing type in tablet record for tablet"..

Reply:
Or even "error changing type in topo record for tablet"

Reply:
Fixed